docs/customizations/playground-examples.md

```
@@ -0,0 +1,61 @@
+import { useOpenapi } from 'vitepress-openapi/client'
+
+useOpenapi({
+  spec: '/openapi-playground-examples.json',
+})
+</script>
+
+# Playground Examples
+
+When using the Playground, parameter examples are automatically populated from the OpenAPI specification. By default, the Playground uses the `example` or `examples` properties defined in the OpenAPI spec.
+
+### Playground-specific Examples
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)